src/rdsk.c

```diff
@@ -144,6 +145,7 @@ struct RC_PROFILE *search_cache_targets(void)
 					}
 				}
 			}
+			free(maps);
```

Comment:
This feels like it may be in the wrong spot and should be located above and below line 143/144 (after the closing brace) and after the `for (z=0` loop.

Also `if (maps) free(maps)`

Reply:
Yes, you're right, fixed!